### ISSUE TITLE
Problem recovering sd of random intercept in gamma model

### ISSUE THREAD

**Issue by bbolker** [lme4/lme4#557]
https://stats.stackexchange.com/questions/447029/parameter-recovery-gamma-distribution-and-model-with-a-random-intercept#447029

**Comment by nahorp** [comment]
Hello Ben, just wondering if you have had any time to explore this issue?

Thank you,

Rohan

**Comment by bbolker** [comment]
not yet ...

**Comment by nahorp** [comment]
Just to update information from the thread,

The same simulation was run with `glmmTMB` (673 simulations) and `brms` (209 simulations) and the random intercept parameter is being recovered accurately so the issue seems to be specific to `lme4`.

**Comment by nahorp** [comment]
Just a friendly reminder @bbolker :)

**Comment by nahorp** [comment]
Hi @bbolker, just wondering if this issue has been resolved in newer lme4 versions? Thank you.

**Comment by bbolker** [comment]
No.  I've just started digging into it: https://github.com/lme4/lme4/blob/master/misc/GH_557.R

So far nothing is jumping out at me.  It's still possible that this is driven in some way by 'bad' estimation of the `sigma` (residual variation) component, or some mismatch in parameterizations/expectations.  A very simple example shows that we can recover `theta`, which is the *internal* parameter of the random-effects SD (it's actually the *scaled* random-effects SD, so it's possible there's a mismatch somewhere?)

**Comment by bbolker** [comment]
this should be resolved by recent changes (in `Gamma_GLMM` branch, soon to be merged into master)
